Automattic/sensei
# Improve theme support for TwentyTwenty

Fixes #2806

#### Changes proposed in this Pull Request:

* Reset padding for <section> elements
* Fix oversized font for progressbar
* Reset archive page header background & spacing
* Fix sorting option floating outside content area in course/module archive pages
* Add a bit of spacing between "Modules" header and module box

#### Testing instructions:

* Browse through courses frontend with theme TwentyTwenty
* Check that a few other themes continue to look the same

#### Screenshot / Video

Course archive page before/after:

<img width="872" alt="Screenshot 2020-02-13 at 17 30 52" src="https://user-images.githubusercontent.com/176949/74456282-ddae6700-4e86-11ea-8e85-6bccaffa92f2.png">

<img width="872" alt="Screenshot 2020-02-13 at 17 31 55" src="https://user-images.githubusercontent.com/176949/74456209-bd7ea800-4e86-11ea-987d-f5ed57d535db.png">

#### Proposed changelog entry for your changes:

* Improve support for TwentyTwenty theme

#### Changed Templates
`templates/archive-course.php` - Add container around controls and header.
